src/routes/forgotPassword.test.js

```diff
@@ -1,5 +1,116 @@
+import mongoose from 'mongoose'
+import request from 'supertest'
+import crypto from 'crypto'
+import createMongooseMemoryServer from 'mongoose-memory'
+import jwt from 'jsonwebtoken'
+
+import createServer from './index.js'
+
+import Admin from '../models/Admin.js'
+
+const mongooseMemoryServer = createMongooseMemoryServer(mongoose)
+const secrets = process.env.SECRETS.split(' ')
+
 describe('/v1/forgot-password/', () => {
-  test('true', () => {
-    expect(true).toBe(true)
+  let app
```

Comment:
You will need to test the email sending here as well. Basically, you need to test out if there is a proper link in the email that you sent out.